src/coreclr/pal/inc/pal.h

```diff
@@ -4069,30 +4069,7 @@ typedef unsigned int wint_t;
 #endif
 
 #ifndef PAL_STDCPP_COMPAT
-
-#if defined(_DEBUG)
-
-/*++
-Function:
-PAL_memcpy
-
-Overlapping buffer-safe version of memcpy.
```

Comment:
The reason for having this memcpy was that the Linux memcpy is not overlapping buffers safe by definition while the windows one is safe with that. I am not sure if it is still needed by the runtime, but it might be.

Reply:
Ah, we just had an assert added there, so I guess it is ok.

Reply:
Yeah. I saw the comment and then only the asserts so figured it was better to just remove it.